### ISSUE TITLE
question about --enable-legacy

### ISSUE THREAD

**Issue by sezero** [dbry/WavPack#166]
Hi David: Legacy wavpack files (generated by wavpack < 4.0 yes) are
disabled by default and needs `--enable-legacy` configury switch to
enable during library build: Is there a compelling reason for that?
Is it unstable / unreliable? And/or, are such legacy wavpack files
rare to care about? Asking it because I am undecided about enabling
it in SDL_mixer's own vendored libwavpack builds.

**Comment by dbry** [comment]
Hi Ozkan. Version 4.0 of WavPack is now around 20 years old, so any files from before that are exceedingly rare (I have not heard about one for years). And I recommend that anyone still in possession of such files should losslessly transcode them to a modern version because there are many technical issues with them. First, they begin with a WAV RIFF header, so some software misidentifies them and plays loud noise. Also, they are formatted as one contiguous stream, so they are very slow to seek and a single bit error can turn an entire file to unrecoverable noise.

In short, my recommendation would definitely be to _not_ include legacy support in any SDL builds. Even FFmpeg, which attempts to decode anything, has correctly decided not to include WavPack legacy support.

Thanks!

**Comment by sezero** [comment]
OK, thanks!
